tests/testthat/test-model_parameters.aov_es_ci.R

```diff
@@ -102,9 +111,488 @@ if (require("insight") && require("effectsize") && require("testthat") && requir
     expect_equivalent(mp$Omega2_CI_low, c(0.71218, NA, 0.50841, NA, 0.53774, NA), tolerance = 1e-3)
     expect_equivalent(na.omit(mp$Omega2_CI_high), es$CI_high, tolerance = 1e-3)
 
-    expect_equal(colnames(mp), c("Response", "Parameter", "Sum_Squares", "df", "Mean_Square", "F", "p",
-                                 "Omega2_partial", "Omega2_CI_low", "Omega2_CI_high", "Eta2_partial",
-                                 "Eta2_CI_low", "Eta2_CI_high", "Epsilon2_partial", "Epsilon2_CI_low",
-                                 "Epsilon2_CI_high"))
+    expect_equal(colnames(mp), c(
+      "Response", "Parameter", "Sum_Squares", "df", "Mean_Square", "F", "p",
+      "Omega2_partial", "Omega2_CI_low", "Omega2_CI_high", "Eta2_partial",
+      "Eta2_CI_low", "Eta2_CI_high", "Epsilon2_partial", "Epsilon2_CI_low",
+      "Epsilon2_CI_high"
+    ))
+  })
+}
+
+# stricter tests ---------------------------------------------------------
```

Comment:
I think you need to add the `tolerance` argument, so tests don't fail on different systems. Furthermore, there's the new `expect_equivalent()`, which is similar to `expect_equal()`, but ignores attributes in objects.